src/render/Renderer.java

```diff
@@ -8,26 +8,24 @@
 import org.lwjgl.opengl.GL11;
 
 import shadowfox.math.*;
-import traingame.Point;
+import traingame.*;
 import traingame.engine.render.*;
-import traingame.engine.render.SpriteBatch;
-import traingame.engine.render.Texture;
-import traingame.World;
-import traingame.Terrain;
 
 public class Renderer implements IFramebufferSizeListener {
     private static final int HEX_WIDTH = 14;
     private static final int HEX_HEIGHT = 16;
     private static final int HEX_ROW_HEIGHT = 12;
 
+    private static final int STYLE = 0;
```

Comment:
Good idea.  However, the hex highlighting gets broken when changing this to 1.  Probably some other function needs to get passed this info.

Reply:
Good catch, the second style was missing a highlight sprite